website/events/admin.py

```diff
@@ -143,6 +145,29 @@ class EventAdmin(DoNextTranslatedModelAdmin):
         (_("Extra"), {"fields": ("slide", "documents"), "classes": ("collapse",)}),
     )
 
+    def everything_paid(self, obj):
+        return obj.everything_paid
+
+    everything_paid.boolean = True
+
+    def present(self, obj):
+        if not obj.registration_required:
+            return None
+
+        count_present_registrations = (
+            obj.present_registrations.count() if obj.present_registrations else 0
+        )
+        count_unpaid_present_registrations = (
+            obj.unpaid_present_registrations.count()
+            if obj.unpaid_present_registrations
+            else 0
+        )
+        count_participants = obj.participants.count() if obj.participants else 0
```

Comment:
These are highly non-optimised queries my friend.

Reply:
Is it? It is a count on a (filtered) set of registrations...

Reply:
you can look at the bug/speeduppaymentusers branch for inspiration

Reply:
Yeah this is nasty